src/main/java/aquality/selenium/core/utilities/SettingsFileUtil.java

```diff
@@ -0,0 +1,11 @@
+package aquality.selenium.core.utilities;
+
+public class SettingsFileUtil {
+
+    private SettingsFileUtil() {
+    }
+
+    public static Object getValueOrDefault(ISettingsFile settingsFile, String path, Object defaultValue) {
```

Comment:
can we move this in the default method of interface to remove this utility class

Reply:
fixed